cftime plotting fails on upstream-dev

seen in https://dev.azure.com/xarray/xarray/_build/results?buildId=3365&view=logs&jobId=2280efed-fda1-53bd-9213-1fa8ec9b4fa8&j=2280efed-fda1-53bd-9213-1fa8ec9b4fa8&t=175181ee-1928-5a6b-f537-168f7a8b7c2d

```
=========================== short test summary info ============================
FAILED xarray/tests/test_plot.py::TestCFDatetimePlot::test_cfdatetime_line_plot
FAILED xarray/tests/test_plot.py::TestCFDatetimePlot::test_cfdatetime_pcolormesh_plot
FAILED xarray/tests/test_plot.py::TestCFDatetimePlot::test_cfdatetime_contour_plot
```

e.g.
```
=================================== FAILURES ===================================
_________________ TestCFDatetimePlot.test_cfdatetime_line_plot _________________

self = <xarray.tests.test_plot.TestCFDatetimePlot object at 0x7f71d66219d0>

    def test_cfdatetime_line_plot(self):
E       ValueError: setting an array element with a sequence. The requested array would exceed the maximum number of dimension of 1.

/usr/share/miniconda/envs/xarray-tests/lib/python3.8/site-packages/matplotlib/transforms.py:943: ValueError
```

--- a/xarray/tests/test_plot.py
+++ b/xarray/tests/test_plot.py
@@ -2251,7 +2251,7 @@ class TestDatetimePlot(PlotTestCase):
         self.darray.plot.line()
 
 
-@pytest.mark.xfail(reason="Failing on upstream tests asof 2020-07-25")
+@pytest.mark.filterwarnings("ignore:setting an array element with a sequence")
 @requires_nc_time_axis
 @requires_cftime
 class TestCFDatetimePlot(PlotTestCase):

